src/containers/GlossPage/components/ExamplesFieldArray.tsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { IGlossExample } from '@ndla/types-backend/concept-api';
+import { ButtonV2 } from '@ndla/button';
+import { colors } from '@ndla/core';
+import { useTranslation } from 'react-i18next';
+import { FieldArray, useField } from 'formik';
+import styled from '@emotion/styled';
```

Comment:
Veldig pirk, men eg foretrekker at disse er sortert litt anderledes.
```suggestion
import { useTranslation } from 'react-i18next';
import { FieldArray, useField } from 'formik';
import styled from '@emotion/styled';
import { ButtonV2 } from '@ndla/button';
import { colors } from '@ndla/core';
import { IGlossExample } from '@ndla/types-backend/concept-api';

```